### ISSUE TITLE
Accuracy 

### ISSUE THREAD

**Issue by loveswine** [danfenghong/IEEE_TGRS_SpectralFormer#10]
Dear Dr.Hong:
    Thanks a lot for you sharing code. I ran your demo for training on Indian dataset twice, the test accuracy is about 76%, which is not as accuracte as your paper, here are the results and arguments,
![屏幕截图 2022-02-06 151032](https://user-images.githubusercontent.com/53184094/152671083-2aaaa2b7-2579-4aaf-8c52-3a0c4b27cf3c.png)

And i also run your spectralformer model in my training code framework for convenient comparing, but the accuracy is relatively low.
So i look for the difference in codes and have a few questions：①Does the spectralformer sensitive to whitening algorithm? 
②I found the training process is stable and convergent in 100 epochs, why set 300 or 600 epochs ? it may be time  consuming.

**Comment by danfenghong** [comment]
Thank you very much for your interest to our work!
The weight_decay should 5e-3 rather than 0. You can refer to the parameter
selection we provided.
Cheers,
Danfeng

loveswine ***@***.***> 于2022年2月6日周日 15:55写道：
